modules/creating-a-machine-pool-cli.adoc

```diff
@@ -66,6 +67,7 @@ For fault-tolerant worker machine pools, choosing a Multi-AZ machine pool distri
 * A Single-AZ machine pool with one availability zone can have a machine count in multiples of 1, such as 1,2,3,4 and so on.
 ====
 <10> Optional: For machine pools in clusters that do not have Red Hat managed VPCs, you can select additional custom security groups to use in your machine pools. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for security groups in the "Additional resources" section.
+<11> Optional: For Multi-AZ clusters and a Single-AZ cluster, you can select subnet to create a Single-AZ machine pool for BYOVPC cluster.
```

Comment:
I feel like this sentence is missing some articles. Do we select _a_ subnet or _the_ subnet? For _a_ BYOVPC cluster or _the_ BYOVPC cluster?